src/cfgparse-global.c

```diff
@@ -24,6 +24,11 @@
 #include <haproxy/peers.h>
 #include <haproxy/protocol.h>
 #include <haproxy/tools.h>
+#include <haproxy/proto_rhttp.h>
```

Comment:
it looks like they sort this by alphabetical order. Can you move it (and the tcp one, see below) under `haproxy/protocol.h`?